.github/workflows/ci.yml

```diff
@@ -320,6 +306,8 @@ jobs:
       - uses: Swatinem/rust-cache@82a92a6e8fbeee089604da2575dc567ae9ddeaab
       - uses: actions/checkout@v4
       - uses: software-mansion/setup-scarb@v1
+        with:
+          scarb-version: "2.10.0"
```

Comment:
Why are we using a fixed scarb version here? 
Also I guess we should eventually remove this, an issue for this would be good.

Reply:
Not needed, it was needed because of some linting changes (which aren't present anymore). Removed.